core/src/main/java/org/elasticsearch/ingest/PipelineStore.java

```diff
@@ -104,8 +103,10 @@ void innerUpdatePipelines(ClusterState state) {
         for (PipelineConfiguration pipeline : ingestMetadata.getPipelines().values()) {
             try {
                 pipelines.put(pipeline.getId(), factory.create(pipeline.getId(), pipeline.getConfigAsMap(), processorFactoryRegistry));
+            } catch (ElasticsearchParseException e) {
```

Comment:
I don't think this catch logic is needed here? We can just let any exception bubble up here, because this logic is executed when each node receives a new cluster state and is processing the pipeline config.